common/presenters/table/object-to-table-head.js

```diff
@@ -2,12 +2,16 @@ const { pickBy } = require('lodash')
 
 const i18n = require('../../../config/i18n')
 
-function objectToTableHead(schemaEl) {
-  const { head } = schemaEl
-  const prop = head.html ? 'html' : 'text'
+function objectToTableHead({ head } = {}) {
+  if (!head) {
+    return undefined
+  }
+
   return pickBy({
     ...head,
-    [prop]: i18n.t(schemaEl.head[prop]),
+    html: head.html ? i18n.t(head.html) : undefined,
```

Comment:
Not sure why this change was necessary - the dynamic prop seems more readable to me. Is there any reason the `pickBy` is actually necessary?

Reply:
The `pickBy` is meant to whittle away the empty props. However, I agree that the dynamic prop seems more readable.

Reply:
Yep, `pickBy` is being used to ensure we don't end up with something like this:

```
{
  html: undefined,
  text: '',
}
```

The thing I was trying to prevent was an empty `text` property being output if provided with neither `html` or `text`. Ie, it will only translate `html` and `text` if you actually provide them and it will clean out any falsy properties provided.

Reply:
But why should one add both `html` and `text`? Wouldn't that be a self-evident dev error?

Reply:
It's not the both scenario this prevents, its the neither. It's preventing a misleading situation where you provide nothing, but you get a `text` property with an empty string.

As you say, if you don't provide either, it's likely an error, so this shouldn't silently provide you with a `text` property. It should return you back nothing because you haven't provided the correct properties.

Reply:
If we decide we want to prevent dev errors, maybe we should introduce type checks.